### ISSUE TITLE
FluentFormsGlobal.$post(...).then(...).catch is not a function on email confirmation save.

### ISSUE THREAD

**Issue by elmarbransch** [fluentform/fluentform#375]
have installed FluentForms traditionally from the wp backend and configured a form.
I edited the email notice to be sent to an email based on a field of the form.

Then I duplicated the notification and wanted to send the same notification to the admin.
If I edit this second confirmation I will be able to save
1. ajax request to admin-ajax.php looks good, payloads as expected, response is json, http 200 Ok
2. the notification is stored
3. in the console I get an error
```TypeError: FluentFormsGlobal.$post(...).then(...).catch is not a function```
4. The fluentforms UI is stuck in loading-progress, i need to reload the page.

FluentForms Version is 5.1.5

I tried debugging and trace step by step, but I am unable to dig any deeper into this error.

**Comment by nkb-bd** [comment]
Could not recreate it ,can you mention the plugins and your theme name ? Will try to reproduce it.

**Comment by elmarbransch** [comment]
I am sorry: it looks my setup is bit outdated, let me first try to update all updatable components and retry this.

**Comment by HrDelwar** [comment]
@elmarbransch Any update about this issue. Hope it's resolved. We are closing the issue. If it's still happen, please re-open the issue and let we know.

**Comment by elmarbransch** [comment]
ah, sorry, lost track here :) 
I had another installation recently, latest version, for a trade fair invitation. Worked great, also with E-Mails.
Now I looked again and it has the same problem as initially described here.

What I did meanwhile: I installed one of my pro licenses on that site, tried around and decided that I don't need the pro version for the use case and removed the pro license from the site manager again (in my profile).
So it looks like here is the root cause .. looks like some edge case with pro un-licensing.
 
Unfortunately, removing the Pro Addon Package does not solve the problem in the end.
